_posts/2019-06-18-list-strategy.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: posts
+title: "Automatic execution strategy selection in Bazel 0.27"
+authors:
+  - ishikhman
+---
+**tl;dr**: Exciting news! After 0.27 release Bazel will *auto select* a suitable execution strategy if none is provided. Customization and strategy enforcement will still be possible and will be much easier to configure.
+
+## Before Bazel 0.27
+
+It was possible to configure Bazel's execution strategy for a build via flags `--spawn_strategy=`, `--strategy=Mnemonic=` and `--strategy_regexp=Regex=`. This mechanism was quite powerful and widely used, but had some drawbacks:
```

Comment:
what is execution strategy? Explain in a short sentence or link to the doc.

Reply:
added a link